src/AdminTab/AdminTab.tsx

```diff
@@ -26,6 +32,11 @@ export const AdminTab: FunctionComponent<AdminTabProps> = ({
   const postgresqlStatus = usePostgresqlStatus();
   const metabaseStatus = useMetabaseStatus();
 
+  const tableListLodable = useTableListLoadable();
+  const tableList = useCachedLoadable(tableListLodable, []);
+  const tableListIsLoading = useIsLoading(tableListLodable);
+  const highlightImport = !tableListIsLoading && tableList.length === 0;
```

Comment:
`useIsLoading` intentionally delays for 500 milliseconds before saying yes in order to avoid UI flicker. I don't think we want that here. `tableListLodable.state === 'loading'` gives the immediate value.

This whole thing can be simplified to:

```suggestion
  const tableListLodable = useTableListLoadable();
  const highlightImport = tableListLodable.state === 'hasData' && tableListLodable.data.length === 0;
```